core/templates/components/state-editor/state-editor-properties-services/state-property.service.ts

```diff
@@ -53,6 +53,12 @@ export class StatePropertyService<StatePropertyType> {
     private utilsService: UtilsService) {
     this.setterMethodKey = null;
   }
+  // eslint-disable-next-line consistent-return
```

Comment:
Avoid using pragma*

Reply:
ESlint checks require always return type. In getter it's not required. 
Removed.